src/config/loader.ts

```diff
@@ -85,14 +85,18 @@ async function _loadUserConfig(
   // prettier-ignore
   let preset: string | undefined = (configOverrides.preset as string) || process.env.NITRO_PRESET || process.env.SERVER_PRESET
 
+  const _dotenv = opts.dotenv ?? configOverrides.dev;
```

Comment:
Let's keep it little simpler. We only need to default `.env, .env.local` in case of fallback and only in dev (after ??)

Reply:
ok, to create a DotenvOptions object I need a value for cwd though.
I'm not sure that process.cwd() is ok for all the scenarios.